content/en/docs/kitex/Tutorials/basic-feature/fallback.md

```diff
@@ -6,4 +6,233 @@ keywords: ["Kitex", "Fallback"]
 description: Kitex Fallback Introduction and Usage Guide.
 ---
 
-Translation TODO 
+> **Support Version: >= v0.5.0  （go.mod dependencies: [github/cloudwego/kitex](https://github.com/cloudwego/kitex))**
+>
+> Protobuf Generated code version：>=v0.5.0（The version can be found in the header of the generated code file and in the version comments.）
+>
+> Thrift : No specific requirement for the generated code version
+
+## 1.Kitex Fallback Functional Description
+
+
+After an RPC request fails, businesses usually take some fallback measures to ensure a valid response (such as constructing a default response when encountering timeouts or circuit breakers). Kitex's Fallback supports handling all exceptional requests. Additionally, as business exceptions usually return through Resp (BaseResp), Fallback also supports handling Resp.
```

Comment:
'business exceptions **usually** return through Resp' modify '**usually**' to '**maybe**'. 

'business exceptions **maybe** return through Resp

Reply:
Modified